### DISCUSSION TITLE
R32.6.1 deepstream-l4t Docker support

### DISCUSSION THREAD

**Discussion by cpeskin12** [OE4T/meta-tegra#779] [General]
Hi all,

Going through the @ichergui's [L4T-R32.6.1 Test/Validation Sheet](https://docs.google.com/spreadsheets/d/1EWG5QZvtWdk74gz48D4xCbegK-bsqMFJ8XOsyGdUUeI/edit?usp=sharing) with `demo-image-full` on the jetson-xavier-nx-devkit, I'm trying to run the `deepstream-test1` app on the latest [deepstream-l4t samples image](https://ngc.nvidia.com/catalog/containers/nvidia:deepstream-l4t/tags) but am seeing it complain about the TensorRT binaries being 0 size:
```
(gst-plugin-scanner:13): GStreamer-WARNING **: 22:12:36.590: Failed to load plugin '/usr/lib/aarch64-linux-gnu/gstreamer-1.0/libgstnvcompositor.so': /usr/lib/aarch64-linux-gnu/gstreamer-1.0/libgstnvcompositor.so: file too short

(gst-plugin-scanner:13): GStreamer-WARNING **: 22:12:36.860: Failed to load plugin '/usr/lib/aarch64-linux-gnu/gstreamer-1.0/deepstream/libnvdsgst_osd.so': /usr/lib/aarch64-linux-gnu/libnvinfer.so.7: file too short

(gst-plugin-scanner:13): GStreamer-WARNING **: 22:12:36.866: Failed to load plugin '/usr/lib/aarch64-linux-gnu/gstreamer-1.0/deepstream/libnvdsgst_inferserver.so': /usr/lib/aarch64-linux-gnu/libnvinfer.so.7: file too short

(gst-plugin-scanner:13): GStreamer-WARNING **: 22:12:36.897: Failed to load plugin '/usr/lib/aarch64-linux-gnu/gstreamer-1.0/deepstream/libnvdsgst_inferaudio.so': /usr/lib/aarch64-linux-gnu/libnvparsers.so.7: file too short

(gst-plugin-scanner:13): GStreamer-WARNING **: 22:12:36.902: Failed to load plugin '/usr/lib/aarch64-linux-gnu/gstreamer-1.0/deepstream/libnvdsgst_infer.so': /usr/lib/aarch64-linux-gnu/libnvparsers.so.7: file too short

```
This makes sense as it's referencing TensorRT v7 and the L4T-R32.6.1 image only includes TensorRT v8 binaries. 
The `deepstream-l4t` containers don't appear to have gotten a JP4.6 update like `l4t-base`, `l4t-ml`, etc. - do we expect this to come sometime soon? 
Has anyone had any success getting a `deepstream-l4t` container/app up and running on a r32.6.1 image?

**Comment by dwalkes** [comment]
Thanks @cpeskin12 - I suspect this means we'll need a new deepstream container before we can use with JP 4.6.

**Reply by dwalkes** [reply]
Looks like it's in the release notes at https://developer.nvidia.com/jetpack-sdk-46

> Deepstream SDK is a complete analytics toolkit for AI-based multi-sensor processing and video and audio understanding.
> **The next version of NVIDIA DeepStream SDK 6.0 will support JetPack 4.6**

So I think we need to wait for Deepstream 6.0 before we can test this.
